src/generatorprofile.cpp

```diff
@@ -853,7 +853,7 @@ void GeneratorProfile::GeneratorProfileImpl::loadProfile(GeneratorProfile::Profi
                                       "\n";
 
         mInterfaceVersionString = "";
-        mImplementationVersionString = "__version__ = \"0.3.0\"\n";
+        mImplementationVersionString = "__version__ = \"0.3.1\"\n";
```

Comment:
Do we really not want to keep the version numbers to be the same for both C and Python?! I would expect that version to be the version of the generator, not the C or Python generator *per se*.

Reply:
I wouldn't agree, they have different version numbers because they can be changed independently of one another.  It is quite likely that they will be synchronised of course but for this sort of change the version number accurately describes the progression of the generated code.

Reply:
Ok, can't be bothered to argue.